NangPaGo-be/src/main/java/com/mars/NangPaGo/domain/user/controller/UserController.java

```diff
@@ -0,0 +1,34 @@
+package com.mars.NangPaGo.domain.user.controller;
+
+import static com.mars.NangPaGo.common.exception.NPGExceptionType.UNAUTHORIZED_NO_AUTHENTICATION_CONTEXT;
+
+import com.mars.NangPaGo.common.dto.ResponseDto;
+import com.mars.NangPaGo.common.exception.NPGException;
+import com.mars.NangPaGo.domain.user.dto.MyPageDto;
+import com.mars.NangPaGo.domain.user.service.MyPageService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/user")
+@RestController
+public class UserController {
+
+    private final MyPageService myPageService;
+
+    @GetMapping("/mypage")
+    public ResponseDto<MyPageDto> findMypage() {
+        String email = SecurityContextHolder.getContext().getAuthentication().getName();
+
+        if (email.equals("anonymousUser")) {
+            throw new NPGException(UNAUTHORIZED_NO_AUTHENTICATION_CONTEXT);
```

Comment:
static method로 변경할 수 있어요.

```suggestion
            throw NPGExceptionType.UNAUTHORIZED_NO_AUTHENTICATION_CONTEXT.of();
```

Reply:
코드 리뷰 반영해서 고쳤습니다.

https://github.com/MARS-LIKELION/NangPaGo/pull/96/commits/527f5444d4fed85f91f639a791c1b8064ca1d19f
new 연산자로 생성하던 다른 부분도 같이 수정했습니다.